build-system/task-runner/amp-task-runner.js

```diff
@@ -253,12 +295,44 @@ function validateUsage(task, taskName, taskFunc) {
   }
 }
 
+/** Sets up shell command autocomplete. */
+function setupAutoComplete() {
+  log(
+    yellow('Set up shell command autocomplete. Please restart your terminal.')
+  );
+  try {
+    complete.setupShellInitFile();
+  } catch (error) {
+    log(red(error.message));
+    log(
+      red('Autocomplete is supported for'),
+      cyan('bash') + ',',
+      cyan('zsh') + ',',
+      red('and'),
+      cyan('fish')
```

Comment:
Can be nested. Up to you whether it reads better, here and other places
```
red(`Autocomplete is supported for ${cyan('bash')}, ${cyan('zsh')}, and ${cyan('fish')}`)
```

Reply:
Eh, rather not nest colours... who knows what some weird terminal out there will do with that 😅